release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -629,6 +636,22 @@ For further information about enabling more control over the C-states, see xref:
 ==== Support for provisioning IPv6 spoke clusters from dual-stack hub clusters
 With this update, you can provision IPv6 address spoke clusters from dual-stack hub clusters. In a zero touch provisioning (ZTP) environment, the HTTP server on the hub cluster that hosts the boot ISO now listens on both IPv4 and IPv6 networks. The provisioning service also checks the baseboard management controller (BMC) address scheme on the target spoke cluster and provides a matching URL for the installation media. These updates offer the ability to provision single-stack, IPv6 spoke clusters from a dual-stack hub cluster.
 
+[id="ocp-4-14-nw-shiftstack-dual-stack"]
+==== Support for dual-stack networking for {rh-openstack} clusters (Technology Preview)
+
+Dual-stack network configuration is now available for clusters that run on {rh-openstack}. You can configure dual-stack networking during the deployment of a cluster on installer-provisioned infrastructure.
+
+For more information, see xref:../installing/installing_openstack/installing-openstack-installer-custom.adoc#install-osp-dualstack_installing-openstack-installer-custom[Configuring a cluster with dual-stack networking].
+
+[id="ocp-4-14-nw-shiftstack-manage-security-groups"]
+==== Security group management for {rh-openstack} clusters
+
+In {product-title} 4.14, security for clusters that run on {rh-openstack} is enhanced. By default, the OpenStack cloud provider now sets the `manage-security-groups` option for load balancers to `true`, ensuring that only node ports that are required for cluster operation are open. Previously, security groups for both compute and control plane machines were configured to open a wide range of node ports for all incoming traffic.  
+
+You can opt to use the previous configuration by setting the `manage-security-groups` option to `false` in the configuration of a load balancer. 
+
+For clusters that are upgraded to 4.14, you must manually update security group rules that open the deployment to all traffic, i.e. permitting `0.0.0.0/0`.  
```

Comment:
Is this instruction to remove the permissive security group rules? It's not clear to me.

Reply:
@mandre How about:

```suggestion
For clusters that are upgraded to 4.14, you must manually remove permissive security group rules that open the deployment to all traffic. For example, you must remove a rule that permits traffic from `0.0.0.0/0`.  
```

Reply:
Perfect :+1: